src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Random random = new Random();
+        Colours[] colour = Colours.values();
+        int randomIndex = random.nextInt(colour.length - 1);
+
+        return String.valueOf(colour[randomIndex]);
```

Comment:
Use the 'name()' method to get the String representation of enum constants instead of 'String.valueOf()' to ensure the correct representation is returned even if 'toString()' is overridden in the enum.